Documentation/gettingstarted/cni-chaining-azure-cni.rst

```diff
@@ -8,12 +8,71 @@
 Azure CNI
 *********
 
-This guide explains how to set up Cilium in combination with Azure CNI. In this
-hybrid mode, the Azure CNI plugin is responsible for setting up the virtual
-network devices as well as address allocation (IPAM). After the initial
-networking is setup, the Cilium CNI plugin is called to attach BPF programs to
-the network devices set up by Azure CNI to enforce network policies, perform
-load-balancing, and encryption.
+.. note::
+
+   This is not the best option to run Cilium on AKS or Azure. Please refer to
+   :ref:`k8s_install_aks` for the best guide to run Cilium in Azure Cloud.
+   Follow this guide if you specifically want to run Cilium in combination with
+   the Azure CNI in a chaining configuration.
+
+.. include:: cni-chaining-limitations.rst
+
+This guide explains how to set up Cilium in combination with Azure CNI in a
+chaining configuration. In this hybrid mode, the Azure CNI plugin is
+responsible for setting up the virtual network devices as well as address
+allocation (IPAM). After the initial networking is setup, the Cilium CNI plugin
+is called to attach eBPF programs to the network devices set up by Azure CNI to
+enforce network policies, perform load-balancing, and encryption.
```

Comment:
Some of these changes do not match the original commits. This workflow for Azure might happen to work but we never included or tested such instructions for v1.8 versions as far as I know. Have you tried these out?

Reply:
Maybe not this particular paragraph, but the bits below.

Reply:
I just applied the patch, didn't look at the context very much...

Reply:
Seems like there was maybe a conflict if this text was inadvertently introduced? In such cases it's also good to highlight in the backport PR description message that there was a conflict, so the original authors of those commits can know to take a closer look.

For this case, I'd suggest just doing a `git show` for the original commit (as pointed out in the backport commit message), and just check that the diff looks the same as the backport commit and remove any bits that don't match. Alternatively if you're up for it, you could try running through the full instructions here to make sure that the full changes work.

Reply:
> In such cases it's also good to highlight in the backport PR description message that there was a conflict, so the original authors of those commits can know to take a closer look.

IMO, all authors of backported commits with conflicts should be requested as reviewers. We would probably need to make that clearer in the docs.

Reply:
I thought mention in the PR description was sufficient to get author's attention generally...

Reply:
There's two things I see here:
1. Is the author notified that their PR is being backported?
1. How does the author evaluate that the change makes sense on this branch?

My feedback above is intended to address (2). If the cherry-pick is smooth and CI is passing then in all honesty I am probably not going to pay a lot of attention unless I happen to know ahead of time that the backport is error-prone. I already did the work and there was no problem with cherrypick, what useful feedback can I provide on my own change?

If instead, it turns out that the backport required resolving conflicts with the older branch, then it's very useful to know this so I can pay closer attention to the area that the backporter highlighted, to double-check that the changes were equivalent on this branch. In the end, the workflow I would use in this case is to pull up the two commits side-by-side (the original and the backport) and check line-by-line that the changes are equivalent. Strictly speaking I think the backporter can and should do this, but the backporter may not have the context to understand the implications of any differences in the code between the branches so that's why I think original author review can catch things that might otherwise be missed, and actually get value out of the review process.

Reply:
@joestringer thanks, this is definitely very helpful. I was doing this a bit too late in the day, and happy to face too many unlucky commits...